selenium/src/test/java/com/epam/tc/hw2/ex2/Exercise2Test.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.tc.hw2.ex2;
+
+import com.epam.tc.hw2.BaseChromeTest;
+import java.time.Duration;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Exercise2Test extends BaseChromeTest {
+    public SoftAssertions softly = new SoftAssertions();
+
+    /**
+     * Assert Browser title.
+     */
+    @Test(groups = {"HW2"})
+    public void exercise2() {
+        softly.assertThat(driver.getTitle()).as("Browser title is incorrect").isEqualTo("Home Page");
+
+        /* Perform login. */
+        new WebDriverWait(driver, Duration.ofSeconds(5))
+                .until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("li.uui-profile-menu"))).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+
+        /* Assert Username is loggined.*/
+        softly.assertThat(driver.findElement(By.id("user-name")).getText()).as("Roman Iovlev is not logged")
+                .isEqualTo("ROMAN IOVLEV");
+
+        /* Open through the header menu Service -> Different Elements Page. */
+        driver.findElement(By.cssSelector("ul.m-l8>li>a.dropdown-toggle")).click();
+        driver.findElement(By.xpath("//a[text()='Different elements']")).click();
+        softly.assertThat(driver.getTitle()).isEqualTo("Different Elements").as("Browser title is incorrect");
+
+        /* Select checkboxes. */
+        driver.findElement(By.xpath("//label[text()[contains(., " + "' Water')]]/*[@type='checkbox']"))
+                .click();
+        softly.assertThat(driver.findElement(By.xpath("//label[text()[contains(., "
+                + "' Water')]]/*[@type='checkbox']")).isSelected()).isTrue().as("Checkbox is not selected");
```

Comment:
Try to avoid splitting selectors on different lines. 
This is better to maintain because you can copy it, check it in the browser, and fix it easily:
```java
By.xpath("//label[text()[contains(., ' Water')]]/*[@type='checkbox']")
```
And usually if selector is too long and requires line wrapping it's probably not the best selector :)